.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: 6KRsCCRjqtti37P140MMCgIDQ8Ftmb9dd
```

Comment:
you're not supposed to check this in -- travis will pass it in via an environment variable

https://coveralls.zendesk.com/hc/en-us/articles/201347419-Coveralls-currently-supports